plenum/server/view_change/view_changer.py

```diff
@@ -184,16 +185,24 @@ def __init__(self, provider: ViewChangerDataProvider):
         # Force periodic view change if enabled in config
         force_view_change_freq = self.config.ForceViewChangeFreq
         if force_view_change_freq > 0:
-            self.startRepeating(self.on_master_degradation, force_view_change_freq)
+            self._timer_master_degradation = RepeatingTimer(self._timer,
```

Comment:
We don't access them here, but theoretically we could (for example to stop and start them again). Another reason for keeping references to objects was to ensure that these objects are not collected by GC, but on the second thought it is already guaranteed as long as timer is running since scheduled closure `self._repeatable_callback` keeps reference to `RepeatingTimer`.